testplan/web_ui/testing/src/Report/BatchReport.js

```diff
@@ -77,23 +80,30 @@ class BatchReport extends React.Component {
       processedReport,
       this.state.filteredReport.filter
     );
+    
+    const firstFailedUID = (
+      filteredReport.report.status === "failed"
+      || filteredReport.report.status === "error"
+    )
```

Comment:
Maybe worth a comment that it is not looking for "non-passed", but explicitly for failed or error. For instance unknown status is exempt here.